Dlp/src/V2/Gapic/DlpServiceGapicClient.php

```diff
@@ -1427,11 +1429,7 @@ public function deleteDeidentifyTemplate($name, $optionalArgs = [])
      *                             Optional.
      *
      *     @type InspectJobConfig $inspectJob
```

Comment:
These descriptions being removed is causing the doc generator to fail. Same for the next argument.

Reply:
Also filed https://github.com/googleapis/toolkit/issues/1915